### ISSUE TITLE
Loading a description with relative paths that don't start with ./

### ISSUE THREAD

**Issue by stephane-caron** [stack-of-tasks/pinocchio#1786]
This is a follow-up https://github.com/stack-of-tasks/pinocchio/issues/1747 on relative paths in URDF.

Checking back on the issue today, I found that this [mini_cheetah_urdf](https://github.com/Derek-TH-Wang/mini_cheetah_urdf) description does not load in Pinocchio. This is what I get with ``package_dirs`` set to the root of the repository:

```python
In [1]: from robot_descriptions.loaders.pinocchio import load_robot_description 
In [2]: load_robot_description("mini_cheetah_description")               
-------------------------------------------------------------------------
ValueError                              Traceback (most recent call last)
...
ValueError: Mesh meshes/mini_abad.obj could not be found.
```

The description loads fine if we manually prefix all mesh paths with ``./``.

Perhaps "meshes/mini_body.obj" is not considered a relative path in [parsers/utils.hpp](https://github.com/stack-of-tasks/pinocchio/blob/c0e7066c86018516abd08332966bc592fb68b0fb/src/parsers/utils.hpp#L103)?

Checks:

* I've checked that package_dirs[0] + "meshes/mini_body.obj exists on my filesystem
* Manually replacing all instances of meshes/ by ./meshes/ in the URDF, the description loads properly

This issue also happens with the minitaur description in [bullet3](https://github.com/bulletphysics/bullet3/tree/master/data/quadruped) or the PR2 description in [robot-assets](https://github.com/ankurhanda/robot-assets).

**Comment by jcarpent** [comment]
Thanks @stephane-caron for reporting this issue.
@fabinsch Could you handle this issue and add a test for it?

**Comment by fabinsch** [comment]
Hello @stephane-caron , I tried your code on my machine and I cannot reproduce your error. Which version of Pinocchio are you using. I am on the current devel and can see this in python 3.9
```
(master_pinocchio) ➜  build git:(devel) ✗ ipython          
Python 3.9.12 | packaged by conda-forge | (main, Mar 24 2022, 23:22:55) 
Type 'copyright', 'credits' or 'license' for more information
IPython 8.3.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import pinocchio

In [2]: pinocchio.__version__
Out[2]: '2.6.12'

In [3]: from robot_descriptions.loaders.pinocchio import load_robot_description
   ...: 
   ...: robot = load_robot_description("mini_cheetah_description")

In [4]: robot.visual_model
Out[4]: 
Nb geometry objects = 13
Name: 	 
body_0
Parent frame ID: 	 
2
Parent joint ID: 	 
1
Position in parent frame: 	 
  R =
1 0 0
0 1 0
0 0 1
  p = 0 0 0

Absolute path to mesh file: 	 
/home/fschramm/.cache/robot_descriptions/mini_cheetah_urdf/meshes/mini_body.obj
Scale for transformation of the mesh: 	 
1 1 1
Disable collision: 	 
0
...
```

**Comment by stephane-caron** [comment]
OK, looks like the previous fix didn't make it to 2.6.10 then? I'm running the latest version available from PyPI:

```
In [1]: import pinocchio                                                 

In [2]: pinocchio.__version__                                            
Out[2]: '2.6.10'
```

**Comment by fabinsch** [comment]
Yes, [here](https://github.com/stack-of-tasks/pinocchio/releases/tag/v2.6.11) we can see that the fix was introduced in Pinocchio v2.6.11. We have do update the PyPI and conda to the latest Pinocchio version.

**Comment by stephane-caron** [comment]
Thanks for checking @fabinsch. Let's close the issue here then. I'll keep track and close https://github.com/robot-descriptions/robot_descriptions.py/issues/15 when 2.6.12 hits PyPI (currently it's only conda that's at 2.6.12).
